src/common/style.js

```diff
@@ -71,7 +71,7 @@ export const mix = (...styles/*:Array<?Rules>*/)/*:Rules*/ => {
     const composedStyle = Object.assign({}, ...styles);
     // @FlowIssue: Flow does not get spread here.
     composedStyle[ID] = id;
-    composedStyles[id/*::.toString()*/] = composedStyle;
+    composedStyles[id.toString()] = composedStyle;
```

Comment:
Same deal as in https://github.com/browserhtml/browserhtml/pull/1093/files#r68305288